Supplementary Resource for Journal Article in Microorganisms (https://www.mdpi.com/journal/microorganisms) 

Ecological Trait-Based Digital Categorization of Microbial Genomes for Denitrification Potential

Raphael D. Isokpehi 1, †*, Yungkul Kim 1,*, Sarah E. Krejci 1, and Vishwa D. Trivedi 1  
1    Oyster Microbiome Project, Bethune-Cookman University, Daytona Beach, FL 32114, USA

†    Current Affiliation: Office of Data Science Strategy, National Institutes of Health, Bethesda, MD 20892, USA 

*   Correspondence: raphael.isokpehi@outlook.com (R.D.I); kimy@cookman.edu (Y.K.) 

Abstract: Microorganisms encode proteins that function in the transformations of useful and harmful nitrogenous compounds in the global nitrogen cycle. The major transformations in the nitrogen cycle are nitrogen fixation, nitrification, denitrification, anaerobic ammonium oxidation, and ammonification. The focus of this report is on the complex biogeochemical process of denitrification, which, in the complete form, consists of a series of four enzyme-catalyzed reduction reactions that transforms nitrate to nitrogen gas. Denitrification is a microbial strain-level ecological trait (characteristic) and the denitrification potential (functional performance) can be inferred from trait rules that rely on the presence or absence of genes for denitrifying enzymes in microbial genomes. Despite the global significance of denitrification and associated large-scale genomic and scholarly data sources, there is lack of datasets and interactive computational tools to investigate microbial genomes according to denitrification trait rules. Therefore, our goal is to categorize archaeal and bacterial genomes by denitrification potential based on denitrification traits defined by rules of enzyme involvement in the denitrification reduction steps. We report the integration of datasets on genome, taxonomic lineage, ecosystem and denitrifying enzymes to provide data investigations context for the denitrification potential of microbial strains. We constructed an ecosystem and taxonomic annotated denitrification potential dataset of 62,624 microbial genomes (866 archaea and 61,758 bacteria) that encode at least one of the 12 denitrifying enzymes in the four-step canonical denitrification pathway. Our 4-digit binary-coding scheme categorized the microbial genomes to one of 16 denitrification traits including complete denitrification trait assigned to 3,280 genomes from 260 bacteria genera. The bacterial strains with complete denitrification potential pattern included Arcobacteraceae strains isolated or detected in diverse ecosystems including aquatic, humans, plants and Mollusca (shellfish). The dataset on microbial denitrification potential and associated interactive data investigations tools can serve as research resources for understanding the biochemical, molecular, physiological and other aspects of microbial denitrification. The microbial denitrification data resources produced in our research can also be useful in identifying microbial strains for synthetic denitrifying communities. 

Keywords: archaea, Arcobacteraceae, bacteria, bioinformatics, data investigations, denitrification, ecological trait, genomes, microbial ecology, KEGG Orthology, synthetic denitrifying communities, visual analytics

 

Visual Analytics Resource: https://public.tableau.com/app/profile/qeubic/viz/microbial_denitrifiers/abstract
